src/context-menu/context-menu.m.css

```diff
@@ -0,0 +1,13 @@
+.root {
+	position: absolute;
+	z-index: 11;
```

Comment:
we need to make these z-indexes higher really, i believe we have some guidance on this somewhere

Reply:
https://github.com/dojo/widgets#z-index-layering

Reply:
I've removed these CSS files, as you mentioned in another comment these styles are handled by the popup now and I just failed to remove the unused file